es6.html

```diff
@@ -1,134 +1,9 @@
-      </ul>
-
-      <p>You can view a more detailed list, including a comparison with other engines, on the <a href="https://kangax.github.io/compat-table/es6/">compat-table</a> project page.</p>
-    </div>
-    </div>
-
-    <div class="faq-item">
-      <h2 class="faq-title">
-         Which ES6 features are behind the --es_staging flag?
-      </h2>
-      <div class="faq-body">
-        <ul class="task-list">
-          <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol"><code>Symbol.toStringTag</code></a> (user-definable results for <code>Object.prototype.toString</code>)</li>
-        </ul>
-      </div>
-    </div>
-
-    <div class="faq-item">
-
-      <h2 class="faq-title">
-        I have my infrastructure set up to leverage the --harmony flag. Should I remove it?
-      </h2>
-      <div class="faq-body">
-        <p>The current behaviour of the <code>--harmony</code> flag on io.js is to enable <strong>staged</strong> features only. After all, it is now a synonym of <code>--es_staging</code>. As mentioned above, these are completed features that have not been considered stable yet. If you want to play safe, especially on production environments, consider removing this runtime flag until it ships by default on V8 and, consequently, on io.js. If you keep this enabled, you should be prepared for further io.js upgrades to break your code if V8 changes their semantics to more closely follow the standard.</p>
-      </div>
-    </div>
-
-    <div class="faq-item">
-
-      <h2 class="faq-title">
-         How do I find which version of V8 ships with a particular version of io.js?
-      </h2>
-      <div class="faq-body">
-        <p>io.js provides a simple way to list all dependencies and respective versions that ship with a specific binary through the <code>process</code> global object. In case of the V8 engine, type the following in your terminal to retrieve its version:</p>
-        <div class="highlight highlight-sh"><pre>iojs -p process.versions.v8</span></pre></div>
-
-        <p>E.g. output: <code>3.31.71.4</code>.</p>
-
-      </div>
-    </div>
-  </div>
-
-  <footer class="content">
-    <nav>
-      <a href="https://github.com/iojs/io.js/issues">GitHub Issues</a><!--
-   --><a href="https://github.com/iojs">GitHub Org</a><!--
-   --><a href="https://webchat.freenode.net/?channels=io.js">IRC Chat</a><!--
-   --><a href="http://logs.libuv.org/io.js/latest">Logs</a><!--
-   --><a href="https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#readme">Governance</a>
-    </nav>
-  </footer>
-
-</body>
-
+  <head>
+    <!-- NOTE FOR FUTURE DEVS: this is a _very_ temporary file
+         while iojs.org shifts server settings. Please remove me. -->
+    <meta http-equiv="refresh" content="0;./public/es6.html">
+    <script>window.location.href='./public/es6.html';</script>
```

Comment:
@rvagg here are the root-directory files which will redirect to the nested public directory as a fallback if this PR was to be put in production early